flowey/flowey_lib_hvlite/src/_jobs/build_test_results_website.rs

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+//! Build the test-results website using npm.
+
+use flowey::node::prelude::*;
+
+flowey_request! {
+    pub struct Params {
+        pub done: WriteVar<SideEffect>,
```

Comment:
Instead of having a done member in this struct, consider replacing it with:
```rust
pub output_dir: WriteVar<PathBuf>,
...
fn process_request(...) {
...
ctx.emit_rust_step(... |ctx| {
    output_dir.claim(ctx);
    move |rt| {
        ...
        rt.write(output_dir, <expected output dir>);
        ...
    }
});
...
```
Then in the consolidate_and_publish_gh_pages node you can add a new ReadVar<PathBuf> member to the params structure, that reads the value that you set in the closure above.